### PR TITLE
ci: adapt linelint for better git history

### PR DESCRIPTION
**Pull request by hezhangjian**
This PR introduces LineLint to the CI pipeline to ensure consistent line endings and prevent trailing whitespace across all files. By enforcing these rules, we achieve the following benefits:

- Cleaner Git History: No more unnecessary changes caused by inconsistent line endings or trailing spaces, resulting in more meaningful diffs.
- Improved Code Quality: Maintaining clean files with no trailing spaces or missing newlines contributes to better code readability and professional standards.

### CODE REVIEW

File: .github/actions/common/exit/action.yml [hunk 1]
```diff
@@ -23,4 +23,3 @@ runs:
         fi
```
Comment (daizhenyu) [inline .github/actions/common/exit/action.yml]: two blank lines

File: .github/workflows/agentcore_integration_test.yml [hunk 2]
```diff
@@ -152,4 +152,3 @@ jobs:
         uses: ./.github/actions/scenarios/agentcore/agentmain
 
 
-
```
Comment (lilai23) [inline .github/workflows/agentcore_integration_test.yml]: more than one blank

Comment (hezhangjian) [reply .github/workflows/agentcore_integration_test.yml]: @lilai23 resolved

File: .github/actions/common/exit/action.yml (hunk 1 above)
Comment (hezhangjian) [reply .github/actions/common/exit/action.yml]: @daizhenyu resolved

**Review by lilai23 — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

**Pull request merged**